mm2src/mm2_test_helpers/src/for_tests.rs

```diff
@@ -208,7 +216,11 @@ impl Mm2TestConf {
                 "coins": coins,
                 "rpc_password": DEFAULT_RPC_PASSWORD,
                 "seednodes": seednodes,
-                "is_watcher": true
+                "is_watcher": true,
+                "wait_for_maker_payment_spend_coefficient": wait_for_maker_payment_spend_coefficient,
+                "refund_start_time_coefficient": refund_start_time_coefficient,
+                "search_interval": search_interval,
+                "wait_for_taker_payment_duration": wait_for_taker_payment_duration
```

Comment:
In https://github.com/KomodoPlatform/atomicDEX-API/pull/1552#discussion_r1037070925 it was requested to create `watcher_conf` object and move all these fields there 🙂 Then, we can use `SwapWatcherConf` struct that will have `#[serde(default = "fn_name")]` for the missing fields case (I would also rename some fields to make them shorter a bit):
```rust
fn one_and_half() -> f64 { 1.5 }

fn three_hundred() -> f64 { 300. }

fn sixty() -> f64 { 60. }

#[derive(Deserialize)]
struct SwapWatcherConf {
    #[serde(default = "one_and_half")]
    wait_for_maker_payment_spend_factor: f64,
    #[serde(default = "one_and_half")]
    refund_start_time_factor: f64,
    #[serde(default = "three_hundred")]
    search_interval: f64,
    #[serde(default = "sixty")]
    wait_for_taker_payment_duration: f64,
}
```